### PR TITLE
Adding Universal Actions to the Adaptive Card-Based Loop Components S…

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**
…ample

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +106,7 @@ The [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDev
...
 |7|Messaging extensions - action preview | Demonstrates how to create a Preview and Edit flow for a Messaging Extension. | [View][msgext-action-preview#cs] | [View][msgext-action-preview#js] ![toolkit-icon](assets/toolkit-icon.png) | [View][msgext-action-preview#python] |[View][msgext-action-preview#java]|[View](/samples/msgext-action-preview/csharp/demo-manifest/msgext-action-preview.zip)
 |8|Link unfurling | Messaging Extension that performs link unfurling. | [View][msgext-link-unfurling#cs] | [View][msgext-link-unfurling#js] ![toolkit-icon](assets/toolkit-icon.png) | [View][msgext-link-unfurling#python] | [View][msgext-link-unfurling#java]|[View](/samples/msgext-link-unfurling/csharp/demo-manifest/msgext-link-unfurling.zip)
+|9|Messaging extensions - Adaptive Card based loop components | Messaging extension with adaptive card-based loop components functionality. |     | [View][msgext-unfurling-ac-loop-components#nodejs] ![toolkit-icon](assets/toolkit-icon.png) |
```
Comment (ChetanSharma-msft) [inline README.md:109]: We have already merged One PR for it. Do we still need to update this?

Comment (ChetanSharma-msft) [reply README.md:109]: |9| Messaging extension - Adaptive Card based loop components | Demonstrates how to create Messaging extension with adaptive card-based loop components. |     | [View][msgext-unfurling-ac-loop-components#nodejs] toolkit-icon |

File: README.md [hunk 2]
```diff
@@ -272,6 +273,7 @@ The [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDev
 [msgext-action-quickstart#js]:samples/msgext-action-quickstart/js
 [msgext-search-quickstart#js]:samples/msgext-search-quickstart/js
 [msgext-search-sso-config#js]:samples/msgext-search-sso-config
+[msgext-unfurling-ac-loop-components#nodejs]:samples/msgext-unfurling-ac-loop-components/nodejs
```
Comment (ChetanSharma-msft) [inline README.md:276]: Please check if this is duplicate

File: samples/msgext-unfurling-ac-loop-components/nodejs/README.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ If you copy and paste a link from `https://github.com/OfficeDev/Microsoft-Teams-
 
 - Microsoft Teams is installed and you have an account
 - [NodeJS](https://nodejs.org/en/)
-- [devtunnel](https://aka.ms/TunnelsCliDownload/win-x64) or [ngrok](https://ngrok.com/) latest version or equivalent tunneling solution
+- [devtunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) or [ngrok](https://ngrok.com/) latest version or equivalent tunnelling solution.
```
Comment (ChetanSharma-msft) [inline samples/msgext-unfurling-ac-loop-components/nodejs/README.md:33]: [dev tunnel]

**Review by ChetanSharma-msft — COMMENTED** [review]
Added my review comments.

File: samples/msgext-unfurling-ac-loop-components/nodejs/README.md (hunk 3 above)
Comment (Harikrishnan-MSFT) [reply samples/msgext-unfurling-ac-loop-components/nodejs/README.md:33]: Fixed

File: README.md (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply README.md]: No @ChetanSharma-msft We missed this file.

File: README.md (hunk 2 above)
Comment (Harikrishnan-MSFT) [reply README.md:276]: Fixed

**Comment by Harikrishnan-MSFT** [comment]
**Build Pipeline**

**Review by shuchauh — APPROVED** [review]
Universal actions are working now for AC-Loop code sample. Approving PR.

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving the PR.

**Comment by ChetanSharma-msft** [comment]
Executed build pipeline: https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5998971228

**Review by Mohammed-MSFT — APPROVED** [review]
Changes Looks fine approving it.

**Comment by Harikrishnan-MSFT** [comment]
build pipeline

https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6011069341

**Pull request merged**